### DISCUSSION TITLE
Data Privacy Policy

### DISCUSSION THREAD

**Discussion by kentykaprog** [schemaorg/schemaorg#3046] [Q&A]
Hello, 

Can you please share how long do you store history and code itself on your platform (e.g. 90 days, etc) and it is possible to reach it using an open link as on [search.google](https://search.google.com/)?

Thank you!

**Comment by thadguidry** [comment]
Not sure I understand the question.  The code or standard for Schema.org is in GitHub and it is versioned and its history retrievable with git commands and for longevity stored forever because of [Artic Vault](https://archiveprogram.github.com/arctic-vault/)
